sympy/physics/control/lti.py

```diff
@@ -1867,6 +2391,35 @@ def __neg__(self):
         neg = -self._expr_mat
         return _to_TFM(neg, self.var)
 
+    def __add__(self, other):
+        if not isinstance(other, LinearTimeInvariant):
+             return NotImplemented
+
+        if other.is_SISO:
+            raise TypeError("Cannot add SISO and MIMO systems together.")
+
+        if not isinstance(other, MIMOParallel):
+            return MIMOParallel(self, other)
+        other_arg_list = list(other.args)
+        return MIMOParallel(self, *other_arg_list)
```

Comment:
If flattening is happening here in `__add__` but not in `__new__` then it won't work if `Parallel` is used directly like `Parallel(tf1, Parallel(tf2, tf3))`.

Is that intentional?